### PR TITLE
Advanced help

### PR DESCRIPTION
**Pull request by Hackante**
<h1>Advanced Help Command</h1>
<h3>Features</h3>

1. You can now get more information about a command like usage or description. If there are aliases you can also see those.
2. There is a list of arguments you can use in commands.

<h3>Usages</h3>

Help for another command:
`+help command`
Help for arguments:
`+help args`

~~_Didn't test it on Beta_~~
Tested ✔️

### CODE REVIEW

**Comment by Hackante** [comment]
What do you want me to change? If I'm correct everything works fine and is able to be merged

**Comment by thewilloftheshadow** [comment]
I didn't realize you were done, sorry

File: commands/bot/help.js [hunk 1]
```diff
@@ -1,32 +1,93 @@
...
+        // help embed
+        let embed = new MessageEmbed().setColor(0x7419b4)
+        let cmd_target
+        // Checking if args[0] is a command
+        if (args[0]) {
+            cmd_target = client.commands.get(args[0].toLowerCase()) || client.commands.find((a) => a.aliases && a.aliases.includes(args[0].toLowerCase()))
+        }
+        // If args[0] is a command, send a specific command card.
+        if (cmd_target) {
+            embed
```
Comment (thewilloftheshadow) [inline commands/bot/help.js:21]: Can you add the icon here too for consistency?

**Review by thewilloftheshadow — CHANGES_REQUESTED** [review]
Looks great, just a couple small adjustments

File: commands/bot/help.js [hunk 2]
```diff
@@ -1,32 +1,93 @@
...
+
+
+Note: Some arguments were left out as they are obvious to understand like \`nickname\`.`
+                )
+        }
+        // if args[0] doesn't exist or it is not a command
+        else {
+            embed
+                .setTitle("About Narrator Bot")
+                .setDescription(
```
[the remark this answers is not in the document]
Comment (Hackante) [reply commands/bot/help.js:78]: edited in vs code

**Comment by Hackante** [comment]
OMG it's finally able to merge!

**Comment by Hackante** [comment]
Changes done and added description and usage for settings.js. Please look over it once again. then it's done

**Review by thewilloftheshadow — APPROVED** [review]

**Pull request merged**